src/messages.rs

```diff
@@ -33,11 +32,13 @@ use routing_table::{Prefix, Xorable};
 use routing_table::Authority;
 use rust_sodium::crypto::{box_, sign};
 use rust_sodium::crypto::hash::sha256;
+use sha3;
 use std::collections::{BTreeMap, BTreeSet, HashSet};
 use std::fmt::{self, Debug, Formatter};
 use std::iter;
 use std::time::Duration;
 use super::QUORUM;
+use tiny_keccak::sha3_256;
```

Comment:
is sha3_256 a function?
if so, I think `use tiny_keccak` is preferred here, then `tiny_keccak::sha3_256()` in the code?